deisctl/backend/fleet/list_units.go

```diff
@@ -82,8 +104,11 @@ func (c *FleetClient) ListUnits() (err error) {
 	}
 
 	for _, us := range unitStates {
-		if strings.HasPrefix(us.Name, "deis-") {
-			states = append(states, us)
+		for _, prefix := range unitPrefixes {
```

Comment:
The only way I could think we could optimize it would be to use `for i := range unitPrefixes { fmt.Println(unitPrefixes[i]) }` which would only allocate the iterator, but then we'd be using it to access the value anyways... meh. The Go std lib uses both ways as well so it's tomato vs tomatoe. I like this approach better, anyways :+1: